docs/examples/tensorflow/demo/nvutils/image_processing.py

```diff
@@ -140,7 +140,7 @@ def __init__(self,
         with tf.device("/gpu:0"):
             self.images, self.labels = daliop(
                 serialized_pipeline=serialized_pipe,
-                shapes=[[batch_size, height, width, 3], []],
+                shapes=[(batch_size, height, width, 3), ()],
```

Comment:
Let us propagate this change to other TF examples as well.

Reply:
Done